projects/01-design-and-usability/project.yml

```diff
@@ -0,0 +1,15 @@
+learningObjectives:
+  - ux/user-centricity/centricity
+  - ux/user-centricity/persistency
+  - ux/research/support
+  - ux/research/documentation
+  - ux/research/sources
+  - ux/research/test
+  - ux/synthesis/patterns
+  - ux/synthesis/features
+  - ux/synthesis/feature-prioritization
+  - ux/product-design/content
+  - ux/product-design/usability
+  - ux/product-design/visual-design
+  - ux/planning/monitor
+  - ux/planning/prioritization
```

Comment:
Tomando en cuenta que desde [el README del proyecto](https://github.com/Laboratoria/bootcamp/tree/main/projects/01-design-and-usability) se desprende:

> [...] Asegurándote de validar esta propuesta con usuarios y proponer un diseño visual acorde a las necesidades de usuarios y negocio para el sitio web que elegiste.

¿No serían parte de este proyecto también los OAs asociados a `comunication`? @lalogf

Reply:
@mfdebian: para agilizar la integración de estos cambios te parece si movemos este comentario a un nuevo issue y lo vemos por separado? Te animas a crear el nuevo issue y etiquetar a @lalogf?

Reply:
va!